frontend/packages/dev-console/src/components/dropdown/ResourceDropdown.tsx

```diff
@@ -141,14 +142,14 @@ class ResourceDropdown extends React.Component<ResourceDropdownProps, State> {
     if (updateSelection) {
       let selectedItem = selectedKey;
       if (
-        (_.isEmpty(sortedList) || !sortedList[this.props.selectedKey]) &&
+        (_.isEmpty(sortedList) || !sortedList[selectedKey]) &&
         allSelectorItem &&
-        allSelectorItem.allSelectorKey !== this.props.selectedKey
+        allSelectorItem.allSelectorKey !== selectedKey
       ) {
         selectedItem = allSelectorItem.allSelectorKey;
       } else if (autoSelect && !selectedKey) {
         selectedItem =
-          this.props.loaded && _.isEmpty(sortedList) && this.props.actionItems
+          loaded && _.isEmpty(sortedList) && this.props.actionItems
```

Comment:
What about `this.props.actionItems` ? shouldn't this be `actionItems` ?

Reply:
Updated it, it should be `actionItems`.